src/preferences/dialog/dlgprefreplaygain.cpp

```diff
@@ -3,24 +3,37 @@
 #include "controlobject.h"
 #include "util/math.h"
 
-#define kConfigKey "[ReplayGain]"
-
-static const int kReplayGainReferenceLUFS = -18;
-
-
-DlgPrefReplayGain::DlgPrefReplayGain(QWidget * parent, UserSettingsPointer  _config)
+namespace {
+const char* kConfigKey = "[ReplayGain]";
```

Comment:
I saw that (nice!) -- it would be nicer if all those parts of the codebase used a shared constant like we do for other features:
https://github.com/mixxxdj/mixxx/blob/master/src/shoutcast/defs_shoutcast.h
https://github.com/mixxxdj/mixxx/blob/master/src/vinylcontrol/defs_vinylcontrol.h
https://github.com/mixxxdj/mixxx/blob/master/src/recording/defs_recording.h

but I won't insist. 


We've already had bugs due to different spellings of COs so I would prefer if new code fixed this. In fact we have one bug in master at the moment which is a [Vinylcontrol] versus [VinylControl] issue (though it's in the skins so it wouldn't be helped by this).